examples/iso/k3s-single-node.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: 1.0
+image:
+  arch: x86_64
+  imageType: iso
+  baseImage: SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso
+  outputImageName: single-node-k3s-no-selinux.iso
+kubernetes:
+  version: v1.28.7+k3s1
+  network:
+    apiVIP: 192.168.100.19
+  nodes:
+    - hostname: node1.suse.com
+      type: server
+  manifests:
+    urls:
+      - https://raw.githubusercontent.com/dbw7/misc/main/k8s-examples/apache-example.yaml
```

Comment:
Maybe a more official manifest is better here? Afraid that Danny's repository might turn private at some point and invalidate this. :D

Reply:
https://k8s.io/examples/application/nginx-app.yaml ?

Reply:
Lol, shit, I didn't even look that closely, I just copied it from our tests. Will fix.